### ISSUE TITLE
Set CameraBarcodeReaderView Heigth 

### ISSUE THREAD

**Issue by viniciusverasdossantos** [Redth/ZXing.Net.Maui#4]
I need to have a ListView and the barcode reader below it so that upon detecting it will add the product to the list. I was unable to resize CameraBarcodeReaderView. Is there a possibility?

**Comment by julianadormon** [comment]
Yes please, this does not seem to be responding to HeightRequested.

**Comment by Filskov-IT** [comment]
Yes HeightRequest isn't working properly. It does not properly scale to layouts either. Would love a fix for this

**Comment by pyramid-johng** [comment]
Seems to be a chronic bug, and i saw a proposed fix in a different ticket.  **Please give this some attention.**

**Comment by Pantheas** [comment]
Is this happening only on Android for you guys?
I am probably facing the same issue, however the sizing works fine on iOS for me.

**Comment by christianrr** [comment]
I get it to work with the following handler override

```
protected override PreviewView CreatePlatformView()
    {
        var result = base.CreatePlatformView();
        result.SetScaleType(PreviewView.ScaleType.FillCenter);
        result.SetClipChildren(true);
        result.SetClipToPadding(true);
        result.ClipToOutline = true;
        return result;
    }
```

**Comment by Jabbah76** [comment]
I have been able to work around this by wrapping the CameraBarcodeReaderView inside a (transparent) Frame.
